### ISSUE TITLE
blinding colors, barely legible

### ISSUE THREAD

**Issue by KolyaKorruptis** [bomsy/brackets-compare#29]
This should be obvious by looking at your own screenshots for this extension.
The first has a black background, the second has a light grey background with light yellow comparison. 
Switching to comparison view basically makes me go blind for a while. *hiss*

Could it use the editor's theme?

**Comment by bomsy** [comment]
Yes! i'm looking into getting that to work. there's already an issue for that.
https://github.com/bomsy/brackets-compare/issues/21 Thanks!

**Comment by KolyaKorruptis** [comment]
Cool. Thank you for the work you're putting into this useful extension.

**Comment by schroef** [comment]
I got the text to be darker, i tried doing the BG color but its fetched from a file in different directory. Perhaps you need to add custom CSS that overrules the other one.

PS When i change text using revert chunk, a other app Source Tree doesnt see these changes? thats weird. But this is different thing than this post

**Comment by schroef** [comment]
I think i got something working here, ive adjusted it to my main theme, Dark Material Theme 2.

I just need to get the text in the chunk better visible
![screen shot 2017-01-15 at 6 44 29 pm](https://cloud.githubusercontent.com/assets/6923008/21966897/d3288bd0-db52-11e6-8139-61bd5bca8371.png)

**Comment by smartsystemdesign** [comment]
@schroef how did you fix it?

**Comment by schroef** [comment]
@smartsystemdesign,
i used the dev tools to see what part of brackets CSS was causing the light background. Then i edited that and restarted. The downside is you need to this each time you update.

See the images, you need to edit codemirror-scroll {backgroundcolor: #f8f8f8}

Open the dev tool, see inthe bottom where it probably says "errors" in red. You can see it in the previous image i posted, if you click that the dev tools open, just like in Chrome. Then by highlighting and opening, i found the corresponding css code for the background color
![screen shot 2018-04-26 at 12 14 08 pm](https://user-images.githubusercontent.com/6923008/39318376-f951e794-494b-11e8-9a8b-5ac573958ea6.png)

On OSX its located here, but you can see that when you hover over the style in the dev tool checker.
![screen shot 2018-04-26 at 12 15 33 pm](https://user-images.githubusercontent.com/6923008/39318393-01360d96-494c-11e8-9be8-40c09c2b0c6b.png)

**Comment by smartsystemdesign** [comment]
Thank you!  I'll try that.
